Extensions/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)Directory.Build.props" />
+  <PropertyGroup>
+    <ExtensionsKeyFile>$(MSBuildThisFileDirectory)Extensions.snk</ExtensionsKeyFile>
+  </PropertyGroup>
+</Project>
```

Comment:
Is this made for future possible changes of settings for projects in ```Extensions``` directory?